AbletonMCP_Remote_Script/handlers/session.py

```
@@ -0,0 +1,1128 @@
+            raise ValueError("No parameters specified")
+        return changes
+    except Exception as e:
+        if ctrl:
+            ctrl.log_message("Error setting link: " + str(e))
+        raise
+
+
+# --- Tuning System ---
+
+
+def get_tuning_system(song, ctrl=None):
```

Ruff (0.15.0)

[warning] Unused function argument: `ctrl`

(ARG001)